egs/ami/s5/local/nnet3/run_tdnn_cleanup.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
```

Comment:
But does this really differ from the standard script, apart from the extra affix, and a couple of extra options?  Why don't you just make some changes to the existing script, instead of add a new one?